android/src/main/java/io/branch/rnbranch/RNBranchModule.java

```diff
@@ -733,12 +715,6 @@ private static Branch setupBranch(Context context) {
         if (!mInitialized) {
             Log.i(REACT_CLASS, "Initializing Branch SDK v. " + BuildConfig.VERSION_NAME);
 
-            RNBranchConfig config = new RNBranchConfig(context);
-
-            if (mUseDebug || config.getDebugMode()) branch.setDebug();
```

Comment:
`setDebug` is deprecated and currently no-op. Should we get rid of `mUseDebug` too?

Reply:
In this class? If it's not used because of this, then definitely.